atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -185,22 +186,29 @@ def __init__(self, tasker, workflow, platforms, build_kwargs,
 
         self.worker_builds = []
 
-    def get_excluded_platforms(self):
-        df_dir = self.workflow.source.get_dockerfile_path()[1]
-        exclude_platforms = set()
-        exclude_path = os.path.join(df_dir, self.EXCLUDE_FILENAME)
-        if os.path.exists(exclude_path):
-            with open(exclude_path) as f:
-                for platform in f:
-                    platform = platform.strip()
-                    if not platform:
-                        continue
-                    exclude_platforms.add(platform)
-
-        return exclude_platforms
-
     def get_platforms(self):
-        return self.platforms - self.get_excluded_platforms()
+        df_dir = self.workflow.source.get_dockerfile_path()[1]
+        excluded_platforms = set()
+        container_path = os.path.join(df_dir, self.CONTAINER_FILENAME)
+        if os.path.exists(container_path):
+            with open(container_path) as f:
+                data = yaml.load(f)
+                if 'platforms' not in data:
+                    return self.platforms
```

Comment:
This case doesn't get covered by unit testing:
https://coveralls.io/builds/12327768/source?filename=atomic_reactor%2Fplugins%2Fbuild_orchestrate_build.py#L197